src/pkg/image/image.go

```diff
@@ -112,7 +112,9 @@ func Init(path string, writable bool) (*Image, error) {
 	if writable {
 		flags = os.O_RDWR
 	}
-	resolvedPath, err := filepath.EvalSymlinks(path)
+
+	abspath, _ := filepath.Abs(path)
```

Comment:
My bad ! I forgot but it may be useful to check error and return it

Reply:
ok no problem